pkg/server/server.go

```diff
@@ -185,10 +187,16 @@ func (c *Server) getHandler(mappers []mapper.Mapper, ec2DescribeQps int, ec2Desc
 			panic(fmt.Sprintf("describeinstancesrole %s is not a valid arn", c.ServerEC2DescribeInstancesRoleARN))
 		}
 	}
+	sess := session.Must(session.NewSession())
+	ec2metadata := ec2metadata.New(sess)
+	instanceRegion, err := ec2metadata.Region()
+	if err != nil {
```

Comment:
Since this falls through on error, is passing empty/nil `instanceRegion` to `token.NewVerifier()` and `ec2provider.New()` safe?

Reply:
I am trying to keep the current behavior as it is. EC2 provider currently updates the region from metadata on a best effort. Same goes for NewVerifier where it skips if it finds any error when trying to get the region sts endpoint.